### ISSUE TITLE
[BUG] Proxmox VE is shown to be debian

### ISSUE THREAD

**Issue by 21st-centuryman** [Macchina-CLI/macchina#276]
**Describe the bug**
Proxmox, an operating system for creating a hosting hypervisor, shows its distribution to be debian instead of proxmox. I haven't run into this problem with other fetch software before. 

**To Reproduce**
1. Create a proxmox vm using their VE iso
2. Build the script and run it

**Expected behavior**
I expected the proxmox to be the distro that the software detected instead of debian.

**Screenshots**
![Screenshot 2023-03-11 at 09 41 28](https://user-images.githubusercontent.com/60197499/224474498-2e21e584-b03b-400a-9bc3-f513354b5862.png)

**`macchina --doctor` output**
![Screenshot 2023-03-11 at 09 42 52](https://user-images.githubusercontent.com/60197499/224474575-d9c153d1-1418-46ad-b6ba-520de9bc58bd.png)

**Comment by 21st-centuryman** [comment]
Ok, thank you I guess I was confused as other fetching programs like neofetch have said proxmox. 

I saw now that it does indeed say debian. I can close this for now as I don't think it's worth it to do a workaround for a single distribution.

**Comment by 21st-centuryman** [comment]
Well proxmox is an operating system.  I virtualized it to test out the software but even when I ran it on my bare metal machine thr problem persisted. I think this has to do with that proxmox still has debian information in `/etc/os-release` so I am not sure this is anything that can be fixed unless you plan to detect operating systems in a different way, I know it worked on neofetch if that helps.

This is definetly something I'll bring up to proxmox so more accurate information can be fetched.
